pkgs/by-name/pi/picom-pijulius/package.nix

```diff
@@ -0,0 +1,27 @@
+{
+  picom,
+  lib,
+  fetchFromGitHub,
+  pcre
+}:
+picom.overrideAttrs (
+  finalAttrs: previousAttrs: {
```

Comment:
we can drop this since it's not used here.

```suggestion
  previousAttrs: {
```

Reply:
No you can't...
You can change it to _: though

Reply:
https://nixos.org/manual/nixpkgs/unstable/#sec-pkg-overrideAttrs
> If only a one-argument function is written, the argument has the meaning of `previousAttrs`.

Reply:
I'm an idiot and was getting the order backwards